tests/blend.py

```diff
@@ -64,164 +74,151 @@ def colors(request):
+    result = clip_image({
+        'image': u16_low_med_high,
         'color': color_white,
         'min': range_all[0],
         'max': range_all[1]
-    }])
+    })
 
-    np.testing.assert_array_equal(expected, result)
+    np.testing.assert_array_equal(expected[0], result[0])
+    np.testing.assert_array_equal(expected[1], result[1])
 
 
-def test_range_high(channel_low_med_high, color_white, range_high):
-    '''Blend an image with one channel, testing high range'''
+def test_clip_image_range_high(u16_low_med_high, color_white, range_high):
+    '''Extract high values from image in channel dictionary'''
 
-    expected = np.uint8([
-        [[0, 0, 0]],
-        [[0, 0, 0]],
-        [[255, 255, 255]]
-    ])
+    expected = (
+        np.array([[0], [0], [1]], dtype=np.float64),
+        np.array([1, 1, 1], dtype=np.float64)
+    )
 
-    result = linear_bgr([{
-        'image': channel_low_med_high,
+    result = clip_image({
+        'image': u16_low_med_high,
         'color': color_white,
         'min': range_high[0],
         'max': range_high[1]
-    }])
+    })
 
-    np.testing.assert_array_equal(expected, result)
+    np.testing.assert_array_equal(expected[0], result[0])
+    np.testing.assert_array_equal(expected[1], result[1])
 
 
-def test_range_low(channel_low_med_high, color_white, range_low):
-    '''Blend an image with one channel, testing low range'''
+def test_clip_image_range_low(u16_low_med_high, color_white, range_low):
+    '''Extract low values from image in channel dictionary'''
 
-    expected = np.uint8([
-        [[0, 0, 0]],
-        [[255, 255, 255]],
-        [[0, 0, 0]]
-    ])
+    expected = (
+        np.array([[0], [1], [1]], dtype=np.float64),
+        np.array([1, 1, 1], dtype=np.float64)
+    )
 
-    result = linear_bgr([{
-        'image': channel_low_med_high,
+    result = clip_image({
+        'image': u16_low_med_high,
         'color': color_white,
         'min': range_low[0],
         'max': range_low[1]
-    }])
+    })
 
-    np.testing.assert_array_equal(expected, result)
+    np.testing.assert_array_equal(expected[0], result[0])
+    np.testing.assert_array_equal(expected[1], result[1])
 
 
-def test_color_white(channel_low_med_high, range_all, color_white):
-    '''Blend an image with one channel, testing white color'''
+def test_make_rgb_color_red(f32_low_med_high, color_red):
+    '''Blend an image with one channel, testing red color'''
 
-    expected = np.uint8([
+    expected = np.array([
         [[0, 0, 0]],
-        [[1, 1, 1]],
-        [[255, 255, 255]]
-    ])
+        [[255 / 65535, 0, 0]],
+        [[1, 0, 0]]
+    ], dtype=np.float32)
 
-    result = linear_bgr([{
-        'image': channel_low_med_high,
-        'color': color_white,
-        'min': range_all[0],
-        'max': range_all[1]
-    }])
+    result = make_rgb(f32_low_med_high, color_red)
 
-    np.testing.assert_array_equal(expected, result)
+    np.testing.assert_array_equal(expected[:, :, 0], next(result))
```

Comment:
Lets use `numpy.testing.assert_allclose` for float comparisons in this and all the other floats comparisons.